staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -318,12 +319,90 @@ func (e *Store) Create(ctx genericapirequest.Context, obj runtime.Object) (runti
 	return out, nil
 }
 
+func (e *Store) CreateInitialized(ctx genericapirequest.Context, obj runtime.Object) (runtime.Object, error) {
```

Comment:
I was planning on doing it, but i don't want to do a goroutine per create.  Still noodling.